cmd/osbuild-composer/config.go

```diff
@@ -35,6 +35,9 @@ type ComposerConfigFile struct {
 		JWTACLFile string `toml:"jwt_acl_file"`
 	} `toml:"composer_api"`
 	WeldrAPI WeldrAPIConfig `toml:"weldr_api"`
+	Logging  struct {
+		logLevel string `toml:"log_level"`
```

Comment:
Maybe we can move this option just to the top-level struct? I know we don't have any options there yet but it doesn't really belong to any particular api. wdyt @teg

Reply:
Sounds good to me.